.environment/soap_service/soap/resource/v2/HRVbuD.rsc

```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<soapResource>
+    <id>HRVbuD</id>
+    <name>UploadedFile-mdprCc</name>
+    <projectId>eUAvaN</projectId>
+    <type>WSDL</type>
+</soapResource>
```

Comment:
This sets up CastleMock in our dev environment to have a project that mocks PA's SOAP service.